src/fastsynth/sygus_parser.cpp

```diff
@@ -969,16 +969,22 @@ void sygus_parsert::generate_invariant_constraints()
 
 void sygus_parsert::NTDef_seq()
 {
-  // it is not necessary to give a syntactic template
-  if(smt2_tokenizer.peek()!=smt2_tokenizert::OPEN)
-    return;
-
-  while(smt2_tokenizer.peek()!=smt2_tokenizert::CLOSE)
+  uint8_t openCount = 0u;
+  while(smt2_tokenizer.peek()!=smt2_tokenizert::CLOSE || openCount)
   {
-    NTDef();
+    switch(smt2_tokenizer.next_token())
```

Comment:
I'm getting this compilation error? I'm on CBMC a33ac4e
~~~
sygus_parser.cpp:975:12: error: 6 enumeration values not explicitly handled in switch: 'NONE', 'END_OF_FILE', 'STRING_LITERAL'... [-Werror,-Wswitch-enum]
    switch(smt2_tokenizer.next_token())
           ^
sygus_parser.cpp:975:12: note: add missing switch cases
    switch(smt2_tokenizer.next_token())
~~~

Reply:
Talk about a pedantic compiler. Please try again. Also, CI for Windows and Mac OS appears broken, but I'll create a separate PR for that after these issues are resolved.